lib/Logger.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
or the other way around, but it would be nice to define services the same way everywhere